download/download_test.go

```diff
@@ -601,14 +706,15 @@ func (t *testFixture) oneShot(ctx context.Context, u Update) {
 }
 
 type testServer struct {
-	t              *testing.T
-	expCode        int
-	expEtag        string
-	expAuth        string
-	bundles        map[string]bundle.Bundle
-	server         *httptest.Server
-	etagInResponse bool
-	longPoll       bool
+	t                         *testing.T
+	expCode                   int
+	expEtag                   string
+	expAuth                   string
+	bundles                   map[string]bundle.Bundle
+	server                    *httptest.Server
+	etagInResponse            bool
+	longPoll                  bool
+	opaVendorMediaTypeEnabled bool
```

Comment:
Why do we need `opaVendorMediaTypeEnabled` ? Can we do without it ?

Reply:
`longPoll` is good. I think opaVendorMediaTypeEnabled should be a rest of some test i have try.